docs/content/en/getting_started/upgrading.md

```diff
@@ -72,6 +72,12 @@ godojo installations
 
 If you have installed DefectDojo on "iron" and wish to upgrade the installation, please see the [instructions in the repo](https://github.com/DefectDojo/godojo/blob/master/docs-and-scripts/upgrading.md).
 
+## Upgrading to DefectDojo Version 2.29.x.
+
+**Breaking Change**
+
+Parameter `enable_auditlog` is not possible to set through System settings anymore. If you set this parameter or you need to set it to `False` (to disable audit logging), set environmental variable `DD_ENABLE_AUDITLOG` to `False`.
```

Comment:
Another env var should be added to the `docker-compose.yml` file in all the containers ran by the django image. This should do the trick
```
DD_ENABLE_AUDITLOG: ${DD_ENABLE_AUDITLOG:-False}
```

Reply:
@Maffooch, is this text which I should add to the mentioned file?

Reply:
Correct. Somewhere in the `environment` blocks for the `uwsgi, celerybeat, celeryworker, and init` containers
https://github.com/DefectDojo/django-DefectDojo/blob/19c4e746a320c2f4ed6b724257cdb4f2a59ff7bd/docker-compose.yml#L50

Reply:
Done